src/promptflow-evals/promptflow/evals/_exceptions.py

```diff
@@ -0,0 +1,85 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+"""This includes enums and classes for exceptions for use in promptflow-evals."""
+
+from enum import Enum
+
+from azure.core.exceptions import AzureError
```

Comment:
Is azure core already a dependency of pf ?

Reply:
Yep: https://github.com/microsoft/promptflow/blob/f0b01d502fb89d615436be27f9bb1c62753dd402/src/promptflow-evals/pyproject.toml#L53

Reply:
I will check on this.